scss/_variables.scss

```diff
@@ -818,6 +818,7 @@ $zindex-tooltip:                    1070 !default;
 
 $nav-link-padding-y:                .5rem !default;
 $nav-link-padding-x:                1rem !default;
+$nav-link-transition:               color .3s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out !default;
```

Comment:
The `color`'s transition duration is different from the others, so I felt my eyes irritated. Perhaps all duration should be `.15s` same as the buttons. If it is fixed, this looks good to me.

BTW, the transition mixin generates unnecessary code for prefer reduce motion when given `null`. I will create a PR to fix this at a later date.